tools/ci/before_install.sh

```diff
@@ -11,6 +11,9 @@ else
   cp config/database.pg.yml config/database.yml
   psql -c "CREATE USER root SUPERUSER PASSWORD 'smartvm';" -U postgres
   export BUNDLE_WITHOUT=development
+
+  which bower || npm install -g bower
+  bower install --allow-root -F --config.analytics=false
```

Comment:
Is this required for every test suite?  For example, I doubt replication test suite needs it.